packages/cardano-services/src/DbSyncProvider/index.ts

```diff
@@ -0,0 +1,17 @@
+import { HealthCheckResponse, Provider } from '@cardano-sdk/core';
+import { Pool } from 'pg';
+
+const HEALTH_CHECK_QUERY = 'SELECT 1';
+
+export abstract class DbSyncProvider implements Provider {
```

Comment:
Nice, but I think this should be at the root in a file named `DbSyncProvider.ts` given there's nothing else in the dir.